### DISCUSSION TITLE
CUSA32137 Like A Dragon Infinite Wealth Ultimate Edition v1.16

### DISCUSSION THREAD

**Discussion by adddaar** [GoldHEN/GoldHEN_Cheat_Repository#419] [Cheat requests]
Can someone do CUSA32137 Like A Dragon Infinite Wealth Ultimate Edition v1.16 cheats ? The one we got is for v1.13 and its not compatible with v1.16 . And the save in locked meaning you can't use the save on older versions for the game, thanks

**Comment by adddaar** [comment]
@Pharaoh2k Pharaoh2k
please ?

**Comment by Imar99Imar** [comment]
Yes, v1.16 please,Thank you!
